### ISSUE TITLE
NVDA+Alt+Arrows shortcuts conflict with IndentNav shortcuts

### ISSUE THREAD

**Issue by mltony** [tsengwoody/Access8Math#34]
This is Tony, I am the author of IndentNav add-on. I just wanted to bring it up to your attention that the latest version of Access8Math seems to have conflicting keyboard shortcuts with IndentNav.
In particular: NVDA+Alt+Left/Right/Down arrows.
Another problem is that for some reason when I try to figure out what action is assigned to these shortcuts, I do that by triggering input help (NVDA+num row 1) and then pressing the desired shortcut (say NVDA+Alt+DownArrow), it doesn't say that this shortcut is assigned to anything, making it harder for users to figure out what's going on. I was looking at your code and it seems to me this might be due to the way you assign shortcuts in `@script` annotation:
```
	@script(gestures=[
		"kb:NVDA+alt+downArrow" if active else "kb:alt+downArrow",
```
I don't know whether NVDA is supposed to work with this kind of if statement, but it seems to me this is likely the reason why NVDA cannot tell me what is the action assigned to this shortcut. Also in input gestures dialog this action doesn't seem to appear either.
Also your documentation doesn't seem to mention that your addon uses these shortcuts.

I also want to mention that I don't know what's the right way to resolve this kind of shortcut conflicts should be. The ecosystem of NVDA addons seems to be growing steadily and conflicts like this are inevitable. So I am not necessarily asking you to change your shortcuts if you think they are essential to your addon. But what I would like to ask you is to make sure that these shortcuts can be detected by NVDA properly, that is in Input Help mode they would detect as belonging to your add-on. Otherwise I expect to see many users of IndentNav coming to me complaining that IndentNav stopped working and unable to debug this problem on their own.
Thanks!

**Comment by tsengwoody** [comment]
Input help not show these shortcuts are not caused by the if statement.
The reason is that I did not provide description parameter.
I will add description parametersetting to solve this problem.
If it is necessary to avoid the conflict between Access8Math and IndentNav shortcut keys, it is recommended to uncheck "Using NVDA+gesture to active action" in General settings.
This option is used to solve the issue #33 problem, but addition of the NVDA key causes conflicts with IndentNav.
I have been using IndentNav. This is why I used alt+leftarrow/rightarrow as a shortcut originally to Avoid conflict with IndentNav.
I think of ways to avoid shortcut conflicts as much as possible when designing shortcut.

**Comment by CyrilleB79** [comment]
Also NVDA+shift+Space and many others have no description. Please provide it.

While at checking the scripts descriptions could you also make them with more details. For example, the script `script_shortcut_switch`:
* Current description says: "shortcut switch"
* A possible description with more details could be the one in the add-on's documentation, i.e.: "Turn on or off the shortcut mode."
This would be really helpful for users but also for translators.
Thanks!

At last, the issue of shortcut conflicts is discussed in #32, and more specifically in https://github.com/tsengwoody/Access8Math/issues/32#issuecomment-921863285.

**Comment by tsengwoody** [comment]
Access8Math 3.1 implement Dynamic gesture binding in math writing scenario.
You can activate/deactivate write/block navigate/shortcut gesture by toggle gesture.
This toggle gesture can be changed in input gesture.
